packages/merge/src/components/MergeSummary/MergeSummary.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Typography} from 'tocco-ui'
+import {intlShape} from 'react-intl'
+import _groupBy from 'lodash/groupBy'
+
+const MergeSummary = ({mergeResponse, intl}) => {
+  const notCopiedRelations = Object.entries(_groupBy(mergeResponse.notCopiedRelations, e => e.entityLabel))
+    .map(([entityLabel, list]) => `${entityLabel} (${list.length})`).join(', ')
+
+  const notDeletedEntities = Object.entries(_groupBy(mergeResponse.notDeletedEntities, e => e.entityLabel))
+    .map(([entityLabel, list]) => `${entityLabel} (${list.length})`).join(', ')
+
+  return <>
+    <Typography.P>
+      {intl.formatMessage({id: 'client.merge.summary.success'})}
+      {
+        (mergeResponse.notCopiedRelations.length > 0 || mergeResponse.notDeletedEntities.length > 0)
+        && intl.formatMessage({id: 'client.merge.summary.successHint'})
+      }
+    </Typography.P>
+    {
+      mergeResponse.notCopiedRelations.length > 0
+      && <Typography.P>
+        {intl.formatMessage({id: 'client.merge.summary.notCopiedRelations'})}:<br/>{notCopiedRelations}
+      </Typography.P>
+    }
+    {
+      mergeResponse.notDeletedEntities.length > 0
+      && <Typography.P>
+        {intl.formatMessage({id: 'client.merge.summary.notDeletedEntities'})}:<br/>{notDeletedEntities}
+      </Typography.P>
+    }
+    {
+      mergeResponse.showPermissionMessage
+      && <Typography.P>{intl.formatMessage({id: 'client.merge.summary.permission'})}</Typography.P>
+    }
+  </>
+}
+
+MergeSummary.propTypes = {
+  mergeResponse: PropTypes.shape({
+    notCopiedRelations: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Since we do not have type safety, it's important to have the prop types as specific possible. I would use ProTypes.shape and describe the attributes even further